dotnet-tools.json

```diff
@@ -2,7 +2,7 @@
   "isRoot": true,
   "tools": {
     "dotnet-format": {
-      "version": "4.0.130203",
+      "version": "5.0.140801",
```

Comment:
I am confused on why we are trying to update dotnet-format version in this PR, whose aim is to fix IDE0044 violation coming from CodeStyle analyzer NuGet package. They seem unrelated.

Reply:
The concern is that bringing in a new CodeStyle package brings in a new FormattingAnalyzer which may use a version of the Roslyn Formatter that differs from what is used in dotnet-format. Meaning running dotnet-format may not fix formatting issues that are reported in the IDE.